decidim-meetings/config/locales/en.yml

```diff
@@ -394,6 +394,12 @@ en:
         calendar_url: Calendar URL
         close_window: Close window
         export_calendar: Export calendar
+      close_meeting_reminder_mailer:
+        close_meeting_reminder:
+          body: The meeting '%{meeting_title}' is pending to be closed. Please add a report of the meeting using the "Close meeting" button.
```

Comment:
maybe %{meeting_title} should be a link?

cc @andreslucena , @core/product ?

Reply:
Yes! At least there should be one link to the meeting in the reminder email